src/tools/rest-frontend/resources/assets/js/controllers/main/MainController.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+module.exports = function ($rootScope, $scope, Logger, $state, $anchorScroll, webStructure) {
+
+    var vm = this;
+    $scope.$rootScope = $rootScope;
+
+    // build the dynamic menu
+    $scope.menu = webStructure;
+
+//        vm.currentLanguage = 'de';
```

Comment:
Is this to be commented in when someone wants german as language? Can you give some more comments at places like this? (also for what the important functions do)